scripts/actions/utils/docs-content-tools/i18n-exclusions.yml

```diff
@@ -0,0 +1,6 @@
+excludePath:
+  ja-JP:
```

Comment:
I think we may want to stick with using `jp` to represent the `ja-JP` localization, since that's what writers will be using when they mark a page for human translation. 
```suggestion
  jp:
```

Reply:
I think i just changed that because that's how it was coming through in the object, but it makes more sense to keep it consistent with the way they do it now

Reply:
These are still in their `ja-JP` form. If I was overridden, feel free to ignore this comment.